ui/src/Shared/services/products-listing-service.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { Categories } from '../models/Categories';
+import { Products } from '../models/Products';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProductsListingServiceService {
+
+  constructor(private httpClient: HttpClient) { }
+  apiURL: string = environment.apiUrl;
+  productsUrl: string = `${this.apiURL}/products`;
+  categoriesUrl: string = `${this.apiURL}/categories`;
+  addToCartUrl: string = `${this.apiURL}/addToCart`;
```

Comment:
Use `environment` directly in the above vars

Reply:
Okay. Resolved the issue as suggested.